src/components/EditorSearchBar.js

```diff
@@ -41,22 +41,27 @@ const EditorSearchBar = React.createClass({
   },
 
   componentWillUnmount() {
-    const shortcuts = this.context.shortcuts;
     if (isEnabled("editorSearch")) {
-      shortcuts.off("CmdOrCtrl+F");
+      const shortcuts = this.context.shortcuts;
+      const searchAgainKey = L10N.getStr("sourceSearch.search.again.key");
+      shortcuts.off(`CmdOrCtrl+${L10N.getStr("sourceSearch.search.key")}`);
       shortcuts.off("Escape");
-      shortcuts.off("CmdOrCtrl+Shift+G");
-      shortcuts.off("CmdOrCtrl+G");
+      shortcuts.off(`CmdOrCtrl+Shift+${searchAgainKey}`);
```

Comment:
The line lengths were getting long and this was the same key so I created `searchAgainKey` here and in the DidMount function.